.env.test

```
@@ -0,0 +1,41 @@
+# Configure these with your own Docker registry images
+
+DOCKER_IMAGE_BACKEND=kaapi-guardrails-backend
+
+# Callback Timeouts (in seconds)
+CALLBACK_CONNECT_TIMEOUT=3
+CALLBACK_READ_TIMEOUT=10
+
+# require as a env if you want to use doc transformation
+OPENAI_API_KEY="<ADD-KEY>"
+GUARDRAILS_HUB_API_KEY="<ADD-KEY>"
+AUTH_TOKEN="<ADD-TOKEN>"
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)